foursight_core/react/api/react_api_base.py

```diff
@@ -36,7 +36,7 @@ def create_response(http_status: int = 200,
                         body: Union[dict, list] = None,
                         headers: dict = None,
                         content_type: str = JSON_CONTENT_TYPE) -> Response:
-        if not body:
+        if body is None:
```

Comment:
I'm curious what this is fixing. Was there some other value like `""` or `0` that is falsey that was ending up in the body? I assume most bodies are either `null` or `{}` so this seems like a no-op. Harmless. But puzzling. Maybe add a comment explaining why `None` is being singled out specifically.

Also, and PyCharm should be telling you this, `None` does not satisfy `Union[dict, list]`. I guess you want `Optional[Union[dict, list]]`. Oh, so maybe the list could be `( )` (the empty list)? That would make sense. Yeah, I'd probably just do:

```suggestion
        if body is None:  # unsupplied argument
```

Reply:
It was fixing a case where I was returning an empty *list* but since it was not passing the ``if not body`` in that case it went on to use `{}` which the UI code was not expecting (expecting a list not an object). Will add comments.